### ISSUE TITLE
Need to generate number from image.

### ISSUE THREAD

**Issue by harrysarson** [harrysarson/primeimage#10]
Something like:

```js

function(imageData, params) {
  ...
  return '010295671783842494...';
}

```

See https://developer.mozilla.org/en-US/docs/Web/API/ImageData. Its just a big array.

**Comment by HarrySarson** [comment]
@efosong can I tempt you?

**Comment by efosong** [comment]
Yep I will take a look at some point

**Comment by harrysarson** [comment]
Implemented in 121047cc189fb50398d75a9ae3073523031fc41f
